docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -124,6 +124,13 @@ specification from a Smithy model using a buildscript dependency:
         }
     }
 
+    dependencies {
```

Comment:
This needs to appear in the buildscript too. The buildscript doesn't pull in normal dependencies -- it's isolated from normal dependencies.

Reply:
Added to the buildScript.